src/frontend-pwa/src/views/Eula/eula.styles.tsx

```diff
@@ -1,50 +1,52 @@
+/**
+ * @summary Styling for EULA view.
+ * @author Tyler Maloney
+ * */
+
 import styled from '@emotion/styled';
 import typography from '../../typography';
 
-export const StyledContainer = styled.div`
-    ${typography.toString()}
+export const StyledOuterDiv = styled.div`
     width: 100%;
-    height: 100%;
-    flex: auto;
+    min-height: 100vh;
```

Comment:
Consider the following:

```suggestion
    min-height: 100svh;
```

To account for a potentially improved mobile experience.